docs/_docs/refs/mojito-file-formats.md

```diff
@@ -320,34 +320,34 @@ Source Resource File (English): `resource.xliff`
 
 ```xml
 <?xml version="1.0" encoding="UTF-8"?>
-<xliff xmlns="urn:oasis:names:tc:xliff:document:2.0" version="2.0">
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:okp="okapi-framework:xliff-extensions" version="1.2">
   <file original="" source-language="en" datatype="x-undefined">
     <body>
-      <trans-unit id="1" resname="hello" datatype="php">
+      <trans-unit id="hi">
```

Comment:
how did this get changed?

Reply:
@aurambaj wasnt sure if resname and datatype were part of the 1.2 version so I just left the id prop. Just checked the specs and they do, so I'll place them back